src/app/features/StepDefinitionEditor/Models/MockModel.js

```diff
@@ -40,7 +40,7 @@ var createMockModelConstructor = function (
         var ast = astCreatorService;
 
         var data = {
-            url: ast.literal(new RegExp(this.url))
+            url: ast.literal(new RegExp(this.url.replace(/\?/g,'\\\?'))) //RegExp does not escaping "?", replace all "?" to "\?" in the string before RegExp escaping charactor 
```

Comment:
Put comments on preceding line and add a space after the `//`. Change it to `// The RegExp constructor does not escape special characters, so we need to double-escape "\?" to "\\\?" in the string before creating the RegExp`